packages/components/tabs/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@tradeshift/elements.tabs",
+  "version": "0.4.5",
+  "src": "src/tabs.js",
+  "main": "lib/tabs.umd.js",
+  "browser": "lib/tabs.umd.js",
+  "files": [
+    "lib/*"
+  ],
+  "dependencies": {
+    "@tradeshift/elements": "^0.4.5",
+    "@tradeshift/elements.icon": "file:../icon",
```

Comment:
I think all "file://" links could be replaced with the version number. For local testing you can use the "lerna link" command.